pkg/functions/runner.go

```diff
@@ -156,10 +152,9 @@ func waitFor(job *Job, timeout time.Duration) error {
 	if job.verbose {
 		fmt.Printf("Waiting for %v\n", url)
 	}
-	to := time.After(timeout)
 	for {
 		select {
-		case <-to:
+		case <-time.After(timeout):
```

Comment:
bad rebase?

Reply:
I still see incorrect version.

Reply:
https://github.com/knative/func/pull/1804#discussion_r1224500551
https://github.com/knative/func/pull/1804/commits/14c4c49a14bba1abac4b3f1d8f5074be1361840f

Reply:
Either something is cached somewhere, I missed a different bug, or you think the other way was better :)